Libraries/SPTarkov.Server.Core/Services/LocationLifecycleService.cs

```diff
@@ -77,6 +78,9 @@ protected internal bool IsSide(string playerSide, string sideCheck = Pmc)
     /// </summary>
     public virtual StartLocalRaidResponseData StartLocalRaid(MongoId sessionId, StartLocalRaidRequestData request)
     {
+        // Backup the profile on raid start
+        backupService.Init().Wait();
```

Comment:
```suggestion
        backupService.Init().GetAwaiter().GetResult();
```

This doesn't wrap the exception in an `AggregateException`